tosfs/core.py

```diff
@@ -203,6 +203,87 @@ def info(
 
         return self._try_dir_info(bucket, key, path, fullpath)
 
+    def rmdir(self, path: str) -> None:
+        """Remove a directory if it is empty.
+
+        Parameters
+        ----------
+        path : str
+            The path of the directory to remove. The path should be in the format
+            `tos://bucket_name/directory_path`.
+
+        Raises
+        ------
+        FileNotFoundError
+            If the directory does not exist.
+        NotADirectoryError
+            If the path is not a directory.
+        TosfsError
+            If the directory is not empty,
+             or if there is an error during the removal process.
```

Comment:
If it is a non-empty dir, does it raise an error?

Reply:
yes, it raises an error.